pkg/kubelet/dockershim/network/cni/cni.go

```diff
@@ -68,6 +69,21 @@ type cniPortMapping struct {
 	HostIP        string `json:"hostIP"`
 }
 
+// cniBandwidthEntry maps to the standard CNI bandwidth Capability
+// see: https://github.com/containernetworking/cni/blob/master/CONVENTIONS.md
```

Comment:
Should probably also mention:
https://github.com/containernetworking/plugins/blob/master/plugins/meta/bandwidth/README.md

Reply:
Done.